packages/webawesome/docs/_includes/pro-badge.njk

```diff
@@ -1,5 +1,12 @@
-{% macro proBadge(proBadgeDescription, proBadgeId) %}
-  {% set badgeId = proBadgeId or ("pro-badge-" + ("" | uniqueId(8))) %}
-  <wa-badge appearance="accent" pill class="pro" id="{{ badgeId }}">Pro</wa-badge>
-  <wa-tooltip for="{{ badgeId }}">{{ proBadgeDescription or "This requires access to Web Awesome Pro" }}</wa-tooltip>
+{% macro proBadge(params) %}
```

Comment:
I'll make quick PRs to update `webawesome-pro` and `webawesome-app` instances of this macro to use this new param syntax.